doctr/models/zoo.py

```diff
@@ -9,10 +9,11 @@
 from .recognition import zoo as reco_zoo
 
 
-__all__ = ["ocr_db_sar", "ocr_db_crnn"]
+__all__ = ["ocr_db_sar", "ocr_db_crnn", "ocr_db_sar_resnet"]
 
 default_cfgs: Dict[str, Dict[str, Any]] = {
     'ocr_db_sar': {'detection': 'db_resnet50_predictor', 'recognition': 'sar_vgg16_bn_predictor'},
```

Comment:
Could you rename ocr_db_sar into ocr_db_sar_vgg for clarity?
Also you need to update the documentation and add the new entries in the unittests ;)